README.md

```diff
@@ -1,33 +1,40 @@
 # Welcome
 
-This repository contains the source code for:
+This repository contains the source code & documentation for:
 
 * [Windows Terminal](https://www.microsoft.com/en-us/p/windows-terminal-preview/9n0dx20hk701)
 * The Windows console host (`conhost.exe`)
 * Components shared between the two projects
 * [ColorTool](https://github.com/Microsoft/Terminal/tree/master/src/tools/ColorTool)
 * [Sample projects](https://github.com/Microsoft/Terminal/tree/master/samples) that show how to consume the Windows Console APIs
-  
+
 Other related repositories include:
 
 * [Console API Documentation](https://github.com/MicrosoftDocs/Console-Docs)
 
 ## Installing & running Windows Terminal
 
-> 👉 Note: To run the Windows Terminal you'll need to be running Windows 10 1903 (build 18362) or later
+### Operating System Support
+
+Windows Terminal can be installed on:
+
+* Windows 10 1903 (build 18362) or later
+* [Windows Server 1903](https://docs.microsoft.com/en-us/windows-server/get-started-19/whats-new-in-windows-server-1903) or later (install manually - see "From this repository" below)
 
-### From the Microsoft Store
+### Install via the Microsoft Store
 
-Download the Microsoft Terminal free from the Microsoft Store and it'll be continuously updated:
+On Windows 10, we recommend downloading the Windows Terminal from the Microsoft Store and it'll be continuously updated:
```

Comment:
the grammar here still needs some assistance. Now that it's not a call-to-action ("Download it free!"), the following "and it'll be continuously updated:" is incorrect.

Reply:
Improved the wording. Thanks.